configs/sites/gaea-c5/packages.yaml

```diff
@@ -0,0 +1,241 @@
+        prefix: /lustre/f2/scratch/Dom.Heinzeller/spack-stack-c5/ecflow-5.8.4
+        modules: [ecflow/5.8.4]
+    file:
+      externals:
+      - spec: file@5.32
+        prefix: /usr
+    findutils:
+      externals:
+      - spec: findutils@4.8.0
+        prefix: /usr
+    flex:
+      externals:
+      - spec: flex@2.6.4+lex
+        prefix: /usr
+    gawk:
+      externals:
+      - spec: gawk@4.2.1
+        prefix: /usr
+    gettext:
+      externals:
+      - spec: gettext@0.20.2
+        prefix: /usr
+    ghostscript:
+      externals:
+      - spec: ghostscript@9.52
+        prefix: /usr
+    git:
+      buildable: false
+      externals:
+      - spec: git@2.35.2
+        modules: [git/2.35.2]
+    git-lfs:
+      buildable: false
+      externals:
+      - spec: git-lfs@2.11.0
+        modules: [git-lfs/2.11.0]
+    gmake:
+      externals:
+      - spec: gmake@4.2.1
+        prefix: /usr
+    groff:
+      externals:
+      - spec: groff@1.22.3
+        prefix: /usr
+    hwloc:
+      externals:
+      - spec: hwloc@2.6.0a1
+        prefix: /usr
+    # This package is currently incomplete (no headers), but still works
+    krb5:
+      externals:
+      - spec: krb5@1.16.3
+        #prefix: /usr/lib/mit
+        prefix: /usr
+    libfuse:
+      externals:
+      - spec: libfuse@2.9.7
+        prefix: /usr
+      - spec: libfuse@3.6.1
+        prefix: /usr
+    libtirpc:
+      variants: ~gssapi
+    libtool:
+      externals:
+      - spec: libtool@2.4.6
+        prefix: /usr
+    # This package is currently incomplete (no headers), but still works
+    libxaw:
+      externals:
+      - spec: libxaw@1.10.13
+        prefix: /usr
+    libxml2:
+      externals:
+      - spec: libxml2@2.9.7
+        prefix: /usr
+    # This package is currently incomplete (no headers) and doesn't work
+    # for us. But it's only needed to build libxaw, for which we can use
+    # the existing (incomplete) installation in /usr, see above
+    #libxpm:
+    #  externals:
+    #  - spec: libxpm@4.11.0
+    #    prefix: /usr
+    lustre:
+      externals:
+      - spec: lustre@2.15.0.2_rc2_cray_113_g62287d0
+        prefix: /usr
+    m4:
+      externals:
+      - spec: m4@1.4.18
+        prefix: /usr
+    ncurses:
+      externals:
+      - spec: ncurses@6.1.20180317+termlib abi=6
+        prefix: /usr
+    openjdk:
+      externals:
+      - spec: openjdk@11.0.16_8-suse-150000.3.83.1-x8664
+        prefix: /usr
+    openssh:
+      externals:
+      - spec: openssh@8.4p1
+        prefix: /usr
+    openssl:
+      externals:
+      - spec: openssl@1.1.1d
+        prefix: /usr
+    perl:
+      externals:
+      - spec: perl@5.26.1~cpanm+shared+threads
+        prefix: /usr
+    pkg-config:
+      buildable: false
+      externals:
+      - spec: pkg-config@0.29.2
+        prefix: /usr
+    qt:
+      externals:
+      - spec: qt@5.15.2
+        prefix: /lustre/f2/scratch/Dom.Heinzeller/spack-stack-c5/qt-5.15.2/5.15.2/gcc_64
```

Comment:
ditto

Reply:
ditto ;-)

Reply:
Note that I didn't write the "Maintainers" section for gaea-c5. It's basically a copy&paste from the original gaea, with slightly different modules. I didn't want to do an "official" install yet, since we don't have everything in place to build the full jedi stack (only ncview missing due to X development packages), and there are issues with the metadata servers for the lustre file system f2.